docs/sce_file_contents.md

```diff
@@ -133,17 +132,32 @@ CITE-seq data, when present, is included within the `SingleCellExperiment` as an
 altExp(sce, "CITEseq")
 ```
 
-Within this, the main expression matrix is again found in the `counts` assay, with each column corresponding to a cell or droplet (in the same order as the parent `SingleCellExperiment`) and each row corresponding to an antibody derived tag (ADT).
-Column names are again cell barcode sequences and row names the antibody targets for each ADT.
+Within this, the main expression matrix is again found in the `counts` assay and the normalized expression matrix is found in the `logcounts` assay.
+For each assay, each column corresponds to a cell or droplet (in the same order as the parent `SingleCellExperiment`) and each row corresponds to an antibody derived tag (ADT).
+Column names are again cell barcode sequences and row names are the antibody targets for each ADT.
+
 
 The following additional per-cell data columns for the CITE-seq data can be found in the main `colData` data frame (accessed with `colData(sce)` [as above](#cell-metrics)).
 
 | Column name                | Contents                                          |
 | -------------------------- | ------------------------------------------------- |
-| `altexps_CITEseq_sum  `    | UMI count for CITE-seq ADTs                       |
+| `altexps_CITEseq_sum`      | UMI count for CITE-seq ADTs                       |
 | `altexps_CITEseq_detected` | Number of ADTs detected per cell (ADT count > 0 ) |
 | `altexps_CITEseq_percent`  | Percent of `total` UMI count from ADT reads       |
 
+
+In addition, the following QC statistics from [`DropletUtils::cleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html) can be found in the `colData` of the `"CITEseq"` alternative experiment, accessed with `colData(altExp(sce, "CITEseq"))`.
+
+| Column name                | Contents                                          |
+| -------------------------- | ------------------------------------------------- |
+| `zero.ambient`   | Indicates whether the cell has zero ambient contamination   |
+| `sum.controls` |  The sum of counts for all control features. Only present if negative control ADTs are present. |
+| `high.controls`  | Indicates whether the cell has unusually high total control counts. Only present if negative control ADTs are present.|
+| `ambient.scale` |  The relative amount of ambient contamination. Only present if negative control ADTs are _not_ present. |
+| `high.ambient`  | Indicates whether the cell has unusually high contamination. Only present if negative control ADTs are _not_ present.|
```

Comment:
I just noticed this, and suddenly I don't know how I feel about it! (Not anything I would do differently at our end!)

Reply:
Well I can guarantee you a fun "wait, why are people doing what they are doing?" discussion next Tuesday at journal club to feel all these feelings all over again!